framework/src/main/java/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -499,6 +504,31 @@ public final synchronized void init(ProcessingEnvironment env) {
         }
     }
 
+    /** True if the git.properties file has been printed. */
+    private static boolean gitPropertiesPrinted = false;
+
+    /** Print information about the git repository from which the Checker Framework was compiled. */
```

Comment:
Could we move/keep these support/debugging methods to the bottom of the file? Is there a reason to move them up?

Reply:
It is only called by `init`, so I put it next to `init`.  I have now moved it to the end of the file.